src/store/vaults.ts

```diff
@@ -6,7 +6,11 @@ export type Ratio = {
   denominator: Amount<'nat'>;
 };
 
-export type QuoteAmount = Amount<'set'>;
+export type PriceQuote = {
+  amountIn: Amount<'nat'>;
+  amountOut: Amount<'nat'>;
+  timestamp: bigint;
+};
```

Comment:
[priceAggregatorChainlink publishes `PriceDescription`](https://github.com/Agoric/agoric-sdk/blob/c542cb6eaa52b131c589ad5197373276db6c2452/packages/inter-protocol/src/priceAggregatorChainlink.js#L142) so please import [that](https://github.com/Agoric/agoric-sdk/blob/c542cb6eaa52b131c589ad5197373276db6c2452/packages/zoe/tools/types.js#L16) instead

Reply:
I added `import type { PriceDescription } from '@agoric/zoe/tools/types';` and get `File '/home/samsiegart/dapp-inter/node_modules/@agoric/zoe/tools/types.d.ts' is not a module.`.

I went into types.d.ts and changed `type PriceDescription = {...}` to `export type PriceDescription = {...}` locally and it worked. How can we fix the sdk to export the types by default?

Reply:
Shoot. Well 5f9b5df44822bb794ac219ba00e1f41aa6c56e16 adds a hack, good enough to merge.

> How can we fix the sdk to export the types by default?

That would be more of a mess, creating a lot of surface that can break.

Since the types are structural, it's not horrible to copy them over. But when we do it's use the same name as in agoric-sdk and make a comment pointing to the original. Those can be culled and cleaned up at some point. (I'm not tackling that right now because prices probably don't below in Zoe anyway.)